tools/generator/src/Extensions/Array+Extensions.swift

```diff
@@ -3,12 +3,12 @@ extension Array where Element == String {
     static let validCommandLineArgSuffix: String = "--command_line_args="
 
     func extractCommandLineArguments() -> [String] {
-        return compactMap({ arg -> String? in
+        return flatMap({ arg -> [String] in
             guard arg.hasPrefix(Array.validCommandLineArgSuffix)
             else {
-                return nil
+                return []
             }
-            return String(arg.dropFirst(Array.validCommandLineArgSuffix.count))
+            return String(arg.dropFirst(Array.validCommandLineArgSuffix.count)).components(separatedBy: ",")
```

Comment:
@rasberik Will this work?
![Screenshot 2023-01-06 at 5 37 04 PM](https://user-images.githubusercontent.com/566328/211111601-9efc2108-d2f3-437d-8a7e-c60c3264d4ae.png)